plugins/source/aws/resources/services/backup/jobs.go

```diff
@@ -0,0 +1,44 @@
+package backup
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/backup"
+	"github.com/aws/aws-sdk-go-v2/service/backup/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Jobs() *schema.Table {
+	tableName := "aws_backup_jobs"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/aws-backup/latest/devguide/API_BackupJob.html`,
+		Resolver:    fetchBackupJobs,
+		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "backup"),
+		Transform:   transformers.TransformWithStruct(&types.BackupJob{}, transformers.WithPrimaryKeys("BackupJobId")),
```

Comment:
`BackupJobId` could be enough as a PK based on the API in https://docs.aws.amazon.com/aws-backup/latest/devguide/API_DescribeBackupJob.html (it only accepts `BackupJobId`) but I couldn't find any other docs that confirm it